Description: Refer to original author Robert K. Jung in the manual page.
Author: Guillem Jover <guillem@debian.org>
Origin: vendor
Bug-Debian: https://bugs.debian.org/456275
Forwarded: no
Last-Update: 2008-06-16

Index: b/resource/en/arj.1
===================================================================
--- a/resource/en/arj.1	2005-06-21 21:27:20.000000000 +0300
+++ b/resource/en/arj.1	2008-06-16 08:25:47.000000000 +0300
@@ -21,6 +21,9 @@ arj \- Archiver for .arj files
 .IR archive [ .arj ]
 .RI [ "base directory" ]
 .RI [ "!list name" | "path name" | "wildcard name" ]
+.SH DESCRIPTION
+\fIarj\fP is a compression and file archiving utility. It was invented by
+Robert K. Jung. \fIARJ\fP stands for \fIA\fPrchived by \fIR\fPobert \fIJ\fPung.
 .SH COMMANDS
 .TP
 .B ac
